### PR TITLE
chore: simplify big number (pow10 to shiftedby)

### PR DESCRIPTION
**Pull request by septs**
closes #

image

see https://mikemcl.github.io/bignumber.js/#shift

### CODE REVIEW

**Comment by septs** [comment]
plz, @DimensionDev/qa test

File: packages/mask/src/extension/popups/pages/Wallet/ContractInteraction/index.tsx [hunk 1]
```diff
@@ -283,7 +282,7 @@ const ContractInteraction = memo(() => {
     const tokenPrice = useTokenPrice(chainId, !isNativeTokenInteraction ? token?.address : undefined)
     const nativeTokenPrice = useNativeTokenPrice(nativeToken?.chainId)
     const tokenValueUSD = new BigNumber(tokenAmount)
-        .dividedBy(pow10(tokenDecimals ?? 0))
+        .shiftedBy(-(tokenDecimals ?? 0))
```
Comment (Jack-Works) [inline packages/mask/src/extension/popups/pages/Wallet/ContractInteraction/index.tsx:285]: Why there is a minus sign?

File: packages/mask/src/plugins/Trader/trader/bancor/useTrade.ts [hunk 2]
```diff
@@ -29,8 +28,8 @@ export function useTrade(
     const { BANCOR_ETH_ADDRESS } = useTraderConstants(chainId)
     const user = useAccount()
 
-    const inputAmount = new BigNumber(inputAmountWei).dividedBy(pow10(inputToken?.decimals ?? 0)).toFixed()
-    const outputAmount = new BigNumber(outputAmountWei).dividedBy(pow10(outputToken?.decimals ?? 0)).toFixed()
+    const inputAmount = new BigNumber(inputAmountWei).shiftedBy(-(inputToken?.decimals ?? 0)).toFixed()
```
Comment (Jack-Works) [inline packages/mask/src/plugins/Trader/trader/bancor/useTrade.ts:31]: why some of the code have `-x` but the other doesn't?

File: packages/mask/src/extension/popups/pages/Wallet/ContractInteraction/index.tsx (hunk 1 above)
Comment (septs) [reply packages/mask/src/extension/popups/pages/Wallet/ContractInteraction/index.tsx:285]: see https://mikemcl.github.io/bignumber.js/#shift

**Pull request merged**